### PR TITLE
Update Troubleshooting NFS

### PR DESCRIPTION
**Pull request by bennyz**
- Changed the recommendation for using all_squash in the exports to just "rw"
- Updated to RHEL 7
- Added comments about NFS 3

This article still needs more work to bring it up-to-date with the current distributions

Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by rollandf** [comment]
@nirs  can you please review?

**Comment by bennyz** [comment]
@alhino

File: source/documentation/how-to/troubleshooting/troubleshooting-nfs-storage-issues.html.md [hunk 1]
```diff
@@ -75,21 +76,22 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
...
+     #> groupadd kvm -g 36
+     #> useradd vdsm -u 36 -g kvm
 
-      # mkdir /storage
+     # mkdir /storage
+ 
+     #> chmod 0755 /storage
+     #> chown 36:36 /storage/
+     #> cat /etc/exports
+     /storage    *(rw,sync,no_subtree_check,all_squash,anonuid=36,anongid=36)
```
Comment (mykaul) [inline source/documentation/how-to/troubleshooting/troubleshooting-nfs-storage-issues.html.md]: Why do we have here all_squash and 36:36 ?

File: source/documentation/how-to/troubleshooting/troubleshooting-nfs-storage-issues.html.md [hunk 2]
```diff
@@ -154,7 +168,7 @@ To activate these settings for the running server reload them with
        # sysctl -p 
 
 ### NFS Storage Domain Failure on Fedora 17
```
Comment (dankenigsberg) [inline source/documentation/how-to/troubleshooting/troubleshooting-nfs-storage-issues.html.md]: Why do you keep Fedora 17? It is irrelevant.

**Comment by jirib** [comment]
I doubt adding user and group makes sense but official docs mention it too -- how would i create user and group a on NAS where i have only web ui access? :)

https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.2/html/administration_guide/sect-preparing_and_adding_nfs_storage

**Comment by gregsheremeta** [comment]
@jirib what NAS is it? you'll get better answers by posting your question in the users forum
https://lists.ovirt.org/archives/list/users@ovirt.org/

**Pull request merged**